### PR TITLE
add health check to settings:help section

### PR DESCRIPTION
**Pull request by valpinkman**
Add Health Check button to verify that our services are up and running, and that the user has no problem connecting to them (maybe shielded behind a VPN or something...)

### Type

Feature

### Context

Issue #1349 

### Parts of the app affected / Test plan

Settings

### CODE REVIEW

**Comment by valpinkman** [comment]
Still need to get a verify URL for the secure websocket check (wss://)

File: src/commands/healthCheck.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+type Result = void
+
+const check = async () => {
+  try {
+    await checkCounterValues()
+    await checkManager()
+    await checkBlockchain()
+    await checkSecureWebsockets()
+  } catch (err) {
+    throw err
```
Comment (gre) [inline src/commands/healthCheck.js]: you can remove this `try {} catch` block because it's a noop

File: src/helpers/health.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }
+  } catch (err) {
+    throw new CountervaluesApiFails()
+  }
+}
+
+export const checkSecureWebsockets = async () => {
+  let socket: WebSocketType
+  try {
+    await new Promise((resolve, reject) => {
```
Comment (gre) [inline src/helpers/health.js]: polish simplification but we could directly do this:


```
checkSecureWebsockets  = () => new Promise((resolve, reject) => {
  ...
   reject(new WebsocketConnectionFailed())
  ...
});
```

**Comment by gre** [comment]
we'll rediscuss this feature internally

**Pull request closed without merging**